src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -80,23 +76,26 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
         avatar={handleAvatar()}
         isContinuous={item?.isContinuous}
         message={item?.message}
-        time={calculateTime(item?.updatedAt, true)}
+        time={item?.updatedAt}
         username={handleUserName(item)}
-        type={BaseChatItemTypeProps.MY_ANON_CHAT}
         status={item?.status as ChatStatus}
-        chatType={type}
+        chatType={type ?? ANONYMOUS}
+        messageType={item?.type}
+        data={item?.rawJson}
       />
     );
 
   return (
     <ChatItemTargetText
       key={index}
       avatar={handleAvatar()}
-      isContinuous={item?.isContinuous}
+      isContinuous={item?.isContinuous ?? false}
       message={item?.message}
-      time={calculateTime(item?.updatedAt, true)}
+      time={item?.updatedAt}
       username={handleUserName(item)}
-      type={BaseChatItemTypeProps.ANON_CHAT}
+      chatType={type ?? ANONYMOUS}
+      messageType={item?.type ?? 'regular'}
+      data={item?.rawJson}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `calculateTime(item?.updatedAt, true)` has been replaced with `item?.updatedAt`. This might cause a change in the format of the time displayed in the chat items. If the intention was to display the raw timestamp instead of the calculated time, then this change is fine. Otherwise, you may want to revert this change.
```diff
-        time={calculateTime(item?.updatedAt, true)}
+        time={item?.updatedAt}
```
Also, the `type` prop in both `ChatItemMyText` and `ChatItemTargetText` components has been removed and replaced with `messageType` and `chatType`. Ensure that these new props are correctly handled in the respective components.
```diff
-        type={BaseChatItemTypeProps.MY_ANON_CHAT}
+        messageType={item?.type}
+        chatType={type ?? ANONYMOUS}
```
```diff
-        type={BaseChatItemTypeProps.ANON_CHAT}
+        messageType={item?.type ?? 'regular'}
+        chatType={type ?? ANONYMOUS}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->